### PR TITLE
Pass empty string to productmd instead of None.

### PR DESCRIPTION
**Pull request by simozhan**
JIRA: PDC-1526

### CODE REVIEW

**Comment by bliuredhat** [comment]
:+1:

File: pdc/apps/package/serializers.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ class Meta:
         fields = ('file_name', 'image_format', 'image_type', 'disc_number',
                   'disc_count', 'arch', 'mtime', 'size', 'bootable',
                   'implant_md5', 'volume_id', 'md5', 'sha1', 'sha256',
-                  'composes')
+                  'composes', 'subvariant')
```
Comment (ycheng-aa) [inline pdc/apps/package/serializers.py]: This filed is already in master branch. Adding this looks not necessary for fixing bug. Maybe it's for test?

Comment (simozhan) [inline pdc/apps/package/serializers.py]: OK. fix it. in this case we will not have conflicts.

**Comment by ycheng-aa** [comment]
:+1:

**Pull request merged**